model-optimizer/extensions/front/onnx/gatherelements_ext.py

```diff
@@ -0,0 +1,32 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from extensions.ops.gatherelements import GatherElements
+from mo.front.extractor import FrontExtractorOp
+from mo.front.onnx.extractors.utils import onnx_attr
+
+
+class GatherElementsFrontExtractor(FrontExtractorOp):
+    op = 'GatherElements'
+    enabled = False
```

Comment:
Why disabled?

Reply:
🤦‍♂️ (facepalm) I disabled it mistakenly at the very last moment before pushing. I wanted to disable for op but did for extractor. Here it was true 868ba41c2a09775187ef7447727d0c58842afcb6  Returned back to 'true'.